src/app/features/test-record-summary/test-record-summary.component.html

```diff
@@ -16,7 +16,7 @@ <h3 class="govuk-heading-s">No test records found.</h3>
   </thead>
   <tbody class="govuk-table__body">
     <tr class="govuk-table__row" *ngFor="let testRecord of testRecords">
-      <th scope="row" class="govuk-table__header">{{ testRecord.testStartTimestamp }} (TODO)</th>
+      <th scope="row" class="govuk-table__header">{{ testRecord.testStartTimestamp | date: 'dd/MM/yyyy HH:mm' }}</th>
       <td class="govuk-table__cell">{{ testRecord.vin }} (TODO)</td>
```

Comment:
remove (TODO)

Reply:
Tom is working on this, will be removed by him